rails-commenting.rb

```diff
@@ -56,8 +56,8 @@ def blog_post_params
 
 # app/models/blog_post.rb
 
-# 9)
+# 9) the blog post model class
```

Comment:
Yes, I think I would also elaborate that the class is inheriting from the ApplicationRecord.